darwin/importer/formats/__init__.py

```diff
@@ -1,6 +1,10 @@
+from typing import List
+
+from darwin.types import ImporterFormat
+
 from . import coco, csvtags, csvtagsvideo, darwin, dataloop, pascalvoc
 
-supported_formats = [
+supported_formats: List[ImporterFormat]  = [
```

Comment:
Not sure if this belongs to the scope of this PR (probably not), but I was wondering if `supported_formats` could be turned into a dictionary.

After all, it seems to be used to map a type name to a function.

Reply:
I agree, but for the time being, I feel this PR is already big as it is. 
It definitely grew beyond what I was expecting, as it turns out, it looks like the Typing issue we had with Darwin-py was considerably larger than I initially estimated.

Which means it is a good thing we are working on it now. This work will not only benefit the docs, it has already resulted in several pre-emptive bugfixes. 

As for the dictionary, I defend we create a separate PR for it.